kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryImportServiceImpl.java

```diff
@@ -183,6 +185,11 @@ public boolean importMotherRecord(Map<String, Object> record) {
             return false;
         }
 
+        //validate if it's an updated record compared to one from database
+        if (mother.getUpdatedDateNic() != null && (mctsUpdatedDateNic == null || mother.getUpdatedDateNic().isAfter(mctsUpdatedDateNic))) {
+            return true;
```

Comment:
I drop an update whenever its an older record and we have an updated record in the database.
This is the case when we import within a range of importDates. And why I increment the accepted count in this case is
1)Ideally if the previous import is successful, accepted count would have that older record. 
2)Rejected count has all the hpdviolation check and errors if any in saving the importData. But these older records doesn't fall in this category